.azure-pipelines/cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Temporary script to remove tools from Azure pipelines agent to create more disk space room.
+sudo apt-get update -y || true
+sudo apt-get purge -y --no-upgrade 'ghc-*' 'zulu-*-azure-jdk' 'libllvm*' 'mysql-*' 'dotnet-*' 'libgl1' \
+  'adoptopenjdk-*' 'azure-cli' 'google-chrome-stable' 'firefox' 'hhvm'
```

Comment:
From CI log:
```481078	google-cloud-sdk
317653	temurin-17-jdk
309543	temurin-11-jdk
214576	mongodb-mongosh
195844	temurin-8-jdk
188266	powershell
```

Let's add `temurin-*-jdk` here as well?

Reply:
+1

Maybe purge `mongodb-mongosh` as well, I doubt it's being used on the CI.